phpunit/class-wp-theme-json-test.php

```diff
@@ -797,14 +797,14 @@ function test_remove_invalid_element_pseudo_selectors() {
 
 	function test_get_element_class_name_button() {
 		$expected = 'wp-element-button';
-		$actual   = wp_theme_element_class_name( 'button' );
+		$actual   = wp_theme_get_element_class_name( 'button' );
```

Comment:
Let's actually [revert](https://github.com/WordPress/gutenberg/pull/44099/files#diff-88c997ea8e8ba51f2fa201327e0f1faccd4e892c2fafe33249f72cfd068c72fcL800) this one to `WP_Theme_JSON_Gutenberg::get_element_class_name()`

Quoting https://github.com/WordPress/wordpress-develop/pull/3244:

> Note that I've opted not to replace calls to `WP_Theme_JSON::get_element_class_name` in the [tests](https://github.com/WordPress/wordpress-develop/blob/3381f05fa241daf172d6c1c893fd25959c3b0656/tests/phpunit/tests/theme/wpThemeJson.php#L2945-L2963) -- since those are specifically for the `WP_Theme_JSON` class.

```suggestion
		$actual   = WP_Theme_JSON_Gutenberg::get_element_class_name( 'button' );
```

Reply:
Done!